utils/lit/tests/Inputs/shtest-format/cond_run_requires-present.txt

```diff
@@ -0,0 +1,3 @@
+COND_RUN(a-present-feature): true
```

Comment:
These test files are _inputs_ to tests, you don't have any updates to the FileCheck test to verify that they actually do what you're intending.